test/conformance/platform/urPlatformCreateWithNativeHandle.cpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2022-2023 Intel Corporation
+// Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+// See LICENSE.TXT
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "fixtures.h"
+
+using urPlatformCreateWithNativeHandleTest = uur::platform::urPlatformTest;
+
+TEST_F(urPlatformCreateWithNativeHandleTest, InvalidNullPlatformPointer) {
+    ur_native_handle_t native_handle = nullptr;
+    ASSERT_SUCCESS(urPlatformGetNativeHandle(platform, &native_handle));
```

Comment:
Don't change this as part of this PR - but I think we could have some logic here to skip this test if the entry-point return `UR_RESULT_ERROR_UNSUPPORTED_FEATURE` lots of adapters do not support these entry-points.

Reply:
We actually do something similar is `urDeviceGetInfo` tests

Reply:
Yeah that makes sense, I could go over all the GetNative/CreateWithNative entry points in a separate PR to make that consistent